client/src/index.js

```diff
@@ -10,8 +9,12 @@ class Client {
   }
 
   init() {
-    ReactDOM.render(<App />, this.domElement);
+    if (this.domElement) {
+      ReactDOM.render(<App />, this.domElement);
```

Comment:
what's "this.domElement"?

Reply:
It's 5 loc above in the Client class constructor, what would you suggest to do this better?